docs/pages/index.mdx

```diff
@@ -50,7 +50,10 @@ Teleport is a Certificate Authority and an Access Plane for your infrastructure.
     Create an issue in GitHub.
   </TileListItem>
   <TileListItem href="https://goteleport.com/get-started/">
-    Reach out sales for a demo of Teleport Enterprise.
+    Reach out to sales for a demo of Teleport Enterprise.
+  </TileListItem>
+  <TileListItem href="https://goteleport.com/get-started/">
```

Comment:
Should we really be linking to the same place for both tiles?

Reply:
changed it to the signup link